types/dcp-client/tslint.json

```diff
@@ -1 +1,8 @@
-{ "extends": "@definitelytyped/dtslint/dt.json" }
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "ban-types": false,
+        "redundant-undefined": false,
+        "strict-export-declare-modifiers": false
+    }
```

Comment:
> The linter configuration file, tslint.json should contain { "extends": "@definitelytyped/dtslint/dt.json" }, and no additional rules. [^1]

Why was this config file changed? I don't see a good reason to change it from the required configuration specified in the project's README.

[^1]: https://github.com/DefinitelyTyped/DefinitelyTyped#linter-tslintjson